gprofiler/profilers/node.py

```diff
@@ -35,6 +35,17 @@ class NodeDebuggerUnexpectedResponse(Exception):
     pass
 
 
+class NodeDebuggerProcessUndefined(Exception):
+    pass
+
+
+class ExpectedResult:
```

Comment:
Can be an `enum`, e.g like `class JavaSafemodeOptions(str, Enum)` in `java.py`. If you pass around instances of the enum instead of `int`, you get type checking for free. After all, you don't want others to pass e.g `7777` as the `expected_resutl` value